library/mediawiki

```diff
@@ -2,36 +2,36 @@ Maintainers: David Barratt <dbarratt@wikimedia.org> (@davidbarratt),
              Kunal Mehta <legoktm@debian.org> (@legoktm),
              addshore <addshorewiki@gmail.com> (@addshore)
 GitRepo: https://github.com/wikimedia/mediawiki-docker.git
-GitCommit: d81c6d202933880f4e2ebebe691be34e8b991c61
+GitCommit: fdf347f62615dad789d0e703fc2f6d628c43d4e1
```

Comment:
It looks like https://github.com/wikimedia/mediawiki-docker moved to the `main` branch so this needs a `GitFetch: refs/heads/main`

Reply:
Thanks for pointing that out, fixed in 1ecfc7dabc4ebd54c887af006462a52af7324f41 ✅